rest_framework-stubs/serializers.pyi

```diff
@@ -1,86 +1,29 @@
 from typing import (
-    Dict,
-    Sequence,
-    Optional,
     Any,
+    Callable,
+    Dict,
     Iterable,
+    Iterator,
     List,
-    Callable,
     Mapping,
-    Type,
-    Tuple,
-    NoReturn,
     MutableMapping,
-    Iterator,
+    NoReturn,
+    Optional,
+    Sequence,
+    Tuple,
+    Type,
     Union,
 )
 
-from rest_framework.fields import (
-    Field as Field,
-    CharField as CharField,
-    RegexField as RegexField,
-    EmailField as EmailField,
-    URLField as URLField,
-    FileField as FileField,
-    IntegerField as IntegerField,
-    FloatField as FloatField,
-    BooleanField as BooleanField,
-    NullBooleanField as NullBooleanField,
-    ListField as ListField,
-    DictField as DictField,
-    ChoiceField as ChoiceField,
-    JSONField as JSONField,
-    DateTimeField as DateTimeField,
-    SerializerMethodField as SerializerMethodField,
-    HiddenField as HiddenField,
-    DecimalField as DecimalField,
-    HStoreField as HStoreField,
-    SlugField as SlugField,
-    UUIDField as UUIDField,
-    FilePathField as FilePathField,
-    TimeField as TimeField,
-    MultipleChoiceField as MultipleChoiceField,
-    DateField as DateField,
-    DurationField as DurationField,
-    IPAddressField as IPAddressField,
-    ModelField as ModelField,
-    ReadOnlyField as ReadOnlyField,
-    ImageField as ImageField,
-    SkipField as SkipField,
-    CreateOnlyDefault as CreateOnlyDefault,
-    CurrentUserDefault as CurrentUserDefault,
-)
-from rest_framework.exceptions import (
-    APIException as APIException,
-    ValidationError as ValidationError,
-    ParseError as ParseError,
-    NotAuthenticated as NotAuthenticated,
-    PermissionDenied as PermissionDenied,
-    ErrorDetail as ErrorDetail,
-    AuthenticationFailed as AuthenticationFailed,
-    MethodNotAllowed as MethodNotAllowed,
-    NotAcceptable as NotAcceptable,
-    NotFound as NotFound,
-    Throttled as Throttled,
-    UnsupportedMediaType as UnsupportedMediaType,
-)
-from rest_framework.relations import (
```

Comment:
Basically, we need all deleted re-exports.

Reply:
done

Reply:
done